frontend/src/components/EventSidePanel.js

```diff
@@ -0,0 +1,179 @@
+import React, { Component } from 'react';
+import { Badge, Card, Carousel, Container, Col, Image, Row } from 'react-bootstrap';
+import AccessTimeIcon from '@material-ui/icons/AccessTime';
+import GroupIcon from '@material-ui/icons/Group';
+import StarBorderIcon from '@material-ui/icons/StarBorder';
+import { connect } from "react-redux";
+import CheckCircleOutlinedIcon from '@material-ui/icons/CheckCircleOutlined';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+import PlaceIcon from '@material-ui/icons/Place';
+import moment from 'moment';
+
+const mapStateToProps = state => {
+  return { articles: state.articles };
+}
+
+const CHECK_CIRCLE_ICON_STYLE = {
+  color: "#1CA45C",
+  padding: 0,
+  fontSize: "60px"
+};
+
+const STAR_BORDER_ICON_STYLE = {
+    color: "#ffe733",
+    padding: 0,
+    fontSize: "60px"
+};
+
+class EventSidePanel extends Component {
+  constructor(props) {
+    super(props);
+
+    var info = {};
+
+    this.convertImagesIntoCarouselItems = this.convertImagesIntoCarouselItems.bind(this);
+    this.checkGoing = this.checkGoing.bind(this);
+  }
+
+  convertImagesIntoCarouselItems() {
+    var length = 0;
+    var imageUrl = "";
+    var allImages = "";
+
+    if (this.props.event.imageUrls !== undefined) {
+      length = this.props.event.imageUrls.length;
+    }
+
+    if (length > 0) {
```

Comment:
Should this be greater than 1? Could the line below crash if length is 1? (I really don't know)

Reply:
@Jkim-Hack